src/utility/filter.utility.ts

```diff
@@ -18,6 +18,9 @@ export const filterPath = process.env.FILTER ?? 'app.filter.json';
 export const filters: Array<FilterI> = JSON.parse(read(filterPath) || '[]') as Array<FilterI>;
 
 export function validateTransaction(id: string, tags: Array<Tag>): boolean {
+  if (filters.length === 0) {
```

Comment:
IDK what is this filter used for, but it's empty by default and it results in db not being populated (IDK how is it working in prod however)